library/src/main/scala-new-collections/scala/scalajs/js/WrappedArray.scala

```diff
@@ -21,7 +21,7 @@ import scala.scalajs.js
 
 /** Equivalent of scm.WrappedArray for js.Array */
 @inline
-final class WrappedArray[A](val array: js.Array[A])
+final class WrappedArray[A](array: js.Array[A])
```

Comment:
I wonder these wrapper can be value class, which alocates less garbage...

Reply:
They can't, because they extend non-universal traits.